changelog/@unreleased/pr-4128.v2.yml

```diff
@@ -1,7 +1,6 @@
 type: improvement
 improvement:
-  description: Targeted sweep metrics now expose a histogram `batchSizeHistogram`
-    of the number of entries read in each iteration of targeted sweep, as well as
-    a gauge `batchSizeMean` that samples the mean of that histogram.
+  description: |
+    Targeted sweep metrics now expose a gauge `batchSizeMean` with the mean of the number of entries read in iterations of targeted sweep durinng a sliding time window of one hour.
```

Comment:
nit: `durinng`

Also, the original format is actually allowed - see https://yaml.org/spec/1.2/spec.html#id2788859. Basically

```
key: value 123123123123
ohnoohnoohno
```

is broken, but the additional indent actually makes

```
key: value 123123123123
 ohnoohnoohno
```

acceptable.